src/genegraph/source/graphql/schema/bibliographic_resource.clj

```diff
@@ -27,5 +32,8 @@
                              :path [:dc/date]}
             :abstract {:type 'String
                        :description "The paper's abstract."
-                       :path [:dc/abstract]}}})
+                       :path [:dc/abstract]}
+            :earliest_article {:type 'Boolean
+                               :description "Article represents the earliest article"
+                               :resolve earliest-article}}})
```

Comment:
I don't think this works--earliest article is a fact derived from the assertion and curation, and not a fact about the article. Would remove this.

Reply:
OK. Good reminder not to distribute facts.